Libraries/Text/Text.js

```diff
@@ -30,7 +30,7 @@ var viewConfig = {
   validAttributes: merge(ReactNativeViewAttributes.UIView, {
     isHighlighted: true,
     numberOfLines: true,
-    allowFontScaling: true,
+    allowFontScaling: false,
```

Comment:
what i mean is that validAttributes doesn't control the default value of the props, which you've already done below.